packages/client/src/comp/states/BaseContainer.ts

```diff
@@ -12,14 +12,22 @@ type InputId =
   | 'email'
   | 'firstName'
   | 'lastName'
+  | 'middleName'
+  | 'suffix'
   | 'telephone'
   | 'mailing'
 
 const isInputValid = (id: InputId, value: string): boolean => {
   switch (id) {
     case 'birthdate': return datePattern.test(value)
     case 'firstName':
-    case 'lastName': {
+    case 'lastName':
```

Comment:
This isn't clear.  Handle middle name and Suffix separately to be more clear.  It's a simple enough regex to copy and reuse.